libs/article-parser/src/lib/correlation-score/correlation-score.ts

```diff
@@ -78,29 +60,19 @@ function stemString(input: string) {
 
 function getWholeParagraphCorrelationScore(
   paragraph: string,
-  impacted: string,
-  recommendation: string,
-  impactedSynonyms: string[],
-  recommendationSynonyms: string[]
+  query: string,
+  querySynonyms: string[]
 ): ParsedArticleParagraph {
-  const impactedStem = stemString(impacted);
+  const queryStem = stemString(query);
   const paragraphStemmed = stemString(paragraph);
```

Comment:
can we be consistent with the naming here? (one is named xStem, and one is xStemmed)